crypto/rand/rand_lib.c

```diff
@@ -216,6 +216,12 @@ size_t rand_drbg_get_entropy(RAND_DRBG *drbg,
         }
 
     } else {
+       if (prediction_resistance) {
+            /* We don't have any entropy sources that comply with the NIST standard
+            * to provide prediction resistance. */
+            return 0;
```

Comment:
Well, sigh, after reading NIST SP 800-90C I have to admit that you are right. The notion of prediction resistance is formulated much more precisely there and much stricter than in  NIST SP 800-90C. Since we can't fullfill the requirements, it is consequent to fail.

However, the function should generate an error message for the caller like that:
```
+            /* We don't have any entropy sources that comply with the NIST standard
+            * to provide prediction resistance. */
+            RANDerr(RAND_DRBG_GET_ENTROPY,·PREDICTION_RESISTANCE_NOT_SUPPORTED);
+            return 0;
```

Reply:
Or even better, with reference (please also in commit message).
```
+            /* We don't have any entropy sources that comply with the NIST standard
+             * to provide prediction resistance (see NIST SP 800-90C, Section 5.4).
+             */
+            RANDerr(RAND_DRBG_GET_ENTROPY,·PREDICTION_RESISTANCE_NOT_SUPPORTED);
+            return 0;
```
I will take care of documenting the new `get_entropy` callback signature and add a statement to the manpage that prediction resistance is currently not supported yet.